nvflare/app_common/abstract/model_persistor.py

```diff
@@ -52,3 +53,26 @@ def save_model(self, model: ModelLearnable, fl_ctx: FLContext):
 
         """
         pass
+
+    def get_model_inventory(self, fl_ctx: FLContext) -> {str: ModelDescriptor}:
+        """
+            Get the model inventory of the ModelPersister
+        Args:
+            fl_ctx: FLContext
+
+        Returns: { model_kind: ModelDescriptor }
+
+        """
+        pass
+
+    def get_model(self, model_file, fl_ctx: FLContext) -> object:
+        """
+
+        Args:
+            model_file:
+            fl_ctx:
+
+        Returns:
+
+        """
+        pass
```

Comment:
If we have these inside `ModelPersistor`
Then we don't need `ModelLocator` anymore?

Reply:
ModelLocator has some overlap functions with ModelPersister. But we could expand this concept later. Still good to have them separately.